.gitignore

```diff
@@ -50,3 +50,6 @@ bower.json
 .byebug_history
 
 .idea/
+
+# Ignore encrypted secrets key file.
+config/secrets.yml.key
```

Comment:
what is this file for?

Reply:
nothing, just tested something before. will be removed

Reply:
actually, this is OK ... if you testing something it's ok